parm/soca/ensda/stage_ens_mem.yaml.j2

```diff
@@ -0,0 +1,35 @@
+######################################
+# set some variables
+######################################
+{% set gPDY = previous_cycle | to_YMD %}
+{% set gcyc = previous_cycle | strftime("%H") %}
+{% set PDY = current_cycle | to_YMD %}
+{% set cyc = current_cycle | strftime("%H") %}
+######################################
+# create working directories
+######################################
+mkdir:
+- "{{ DATAenspert }}/ens"
+
+######################################
+# copy ensemble background files
+######################################
+copy:
+{% for mem in range(1, NMEM_ENS + 1) %}
+    # define variables used in the history template
+    {% set tmpl_dict = {'ROTDIR':ROTDIR,
+                        'RUN': GDUMP_ENS,
+                        'YMD':gPDY,
+                        'HH':gcyc,
+                        'MEMDIR':"mem" + '%03d' % mem} %}
+    {% set com_prev_ocn = namespace(COM_OCEAN_HISTORY_MEM = COM_OCEAN_HISTORY_TMPL) %}
+    {% set com_prev_ice = namespace(COM_ICE_HISTORY_MEM = COM_ICE_HISTORY_TMPL) %}
+    {% for key in tmpl_dict.keys() %}
```

Comment:
The latest version of `wxflow` simplifies these find/replace things, see: https://github.com/NOAA-EMC/global-workflow/blob/9658e3d4ebc6118bc231c5e4712077925a1bf296/parm/gdas/snow_stage_ens_update.yaml.j2#L35